src/js/components/TaskFileDownloadComponent.jsx

```diff
@@ -0,0 +1,98 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var MesosActions = require("../actions/MesosActions");
+var MesosEvents = require("../events/MesosEvents");
+var MesosStore = require("../stores/MesosStore");
+
+function matchFileName(name) {
+  return (file) => file.name === name;
+}
+
+var TaskFileLinkComponent = React.createClass({
```

Comment:
Shouldn't this component be called like the filename, ie `TaskFileDownloadComponent` ??